src/template-tags/tickets.php

```diff
@@ -745,272 +765,338 @@ function tribe_tickets_has_meta_fields( $id, $context = null ) {
+			if (
+				! in_array( $post->post_type, $event_types )
+				&& in_array( $mode, $shared_modes )
+			) {
+				$event_id = tribe_tickets_get_event_ids( $post->ID );
 
-			// It will return an array of Events
-			if ( ! empty( $event_id ) ) {
-				$post_id = current( $event_id );
+				// It will return an array of Events
+				if ( ! empty( $event_id ) ) {
+					$post_id = current( $event_id );
+				}
+			} else {
+				return null;
 			}
-		} else {
-			return null;
 		}
-	}
 
-	// Fetch the value
-	$value = get_post_meta( $post_id, $key, true );
+		// Fetch the value
+		$value = get_post_meta( $post_id, $key, true );
 
-	// When dealing with an empty string we assume it's unlimited
-	if ( '' === $value ) {
-		$value = -1;
-	}
+		// When dealing with an empty string we assume it's unlimited
+		if ( '' === $value ) {
+			$value = - 1;
+		}
 
-	return (int) $value;
+		return (int) $value;
+	}
 }
 
-/**
- * Turns a Stock, Remaining or Capacity into a Human Readable Format
- *
- * @since  4.6
- *
- * @param string|int $number Which you are trying to convert.
- * @param string     $mode   Mode this post is on.
- *
- * @return string
- */
-function tribe_tickets_get_readable_amount( $number, $mode = 'own', $display = false ) {
-	$html = array();
+if ( ! function_exists( 'tribe_tickets_get_readable_amount' ) ) {
 
-	$show_parens = Tribe__Tickets__Global_Stock::GLOBAL_STOCK_MODE === $mode || Tribe__Tickets__Global_Stock::CAPPED_STOCK_MODE === $mode;
-	if ( $show_parens ) {
-		$html[] = '(';
-	}
+	/**
+	 * Turns a Stock, Remaining or Capacity into a Human Readable Format
+	 *
+	 * @since  4.6
+	 *
+	 * @param string|int $number Which you are trying to convert.
+	 * @param string     $mode   Mode this post is on.
+	 *
+	 * @return string
+	 */
+	function tribe_tickets_get_readable_amount( $number, $mode = 'own', $display = false ) {
+		$html = array();
 
-	if ( -1 === (int) $number || Tribe__Tickets__Ticket_Object::UNLIMITED_STOCK === $number ) {
-		/** @var Tribe__Tickets__Tickets_Handler $handler */
-		$handler = tribe( 'tickets.handler' );
+		$show_parens = Tribe__Tickets__Global_Stock::GLOBAL_STOCK_MODE === $mode || Tribe__Tickets__Global_Stock::CAPPED_STOCK_MODE === $mode;
+		if ( $show_parens ) {
+			$html[] = '(';
+		}
 
-		$html[] = esc_html( $handler->unlimited_term );
-	} else {
-		$html[] = esc_html( $number );
-	}
+		if ( - 1 === (int) $number || Tribe__Tickets__Ticket_Object::UNLIMITED_STOCK === $number ) {
+			/** @var Tribe__Tickets__Tickets_Handler $handler */
+			$handler = tribe( 'tickets.handler' );
 
-	if ( $show_parens ) {
-		$html[] = ')';
-	}
+			$html[] = esc_html( $handler->unlimited_term );
+		} else {
+			$html[] = esc_html( $number );
+		}
+
+		if ( $show_parens ) {
+			$html[] = ')';
+		}
 
-	$html = implode( '', $html );
+		$html = implode( '', $html );
 
-	if ( true === $display ) {
-		echo $html;
-	}
+		if ( true === $display ) {
+			echo $html;
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$html'

<sub>hash: eb55947318fccba5916c7d632df4bb4c</sub>